article.js

```diff
@@ -26,14 +27,37 @@ CompleteArticle.prototype.calculateDaysAgo = function (date) {
    * from the oject that calls it, and inserts it into the DOM.
    */
 CompleteArticle.prototype.toHTML = function () {
-  var daysAgo = this.calculateDaysAgo(this.publishedOn);
-  var $newClone = $('article').filter(':first').clone();
+  var daysAgo = this.calculateDaysAgo(this.milliDate);
+  var $newClone = $('article:last').clone();
 
   $newClone.find('.articleTitle').text(this.title);
-  //$newClone.find('.author').text('By ' + this.author);
-  $newClone.find('.author').html('<p><a href=' + this.authorUrl + '>By '+ this.author + '</a></p>');
-  $newClone.find('.publishDate').text('Published about ' + daysAgo + ' days ago');
+  $newClone.find('.author').html('<a href= "' + this.authorUrl + '">By '+ this.author + '</a>');
+  $newClone.find('.publishDate').text('Published ' + daysAgo + ' days ago on ' + this.publishedOn);
   $newClone.find('.articleContent').html(this.body);
+  $newClone.find('.category').text(this.category).hide();
+  $newClone.find('.readMore').text('Read More');
+  $newClone.find('.readLess').text('Read Less');
 
   $('article:last').after($newClone);
-};
+
+  /*this.createDropDownFilter(this.author, '#authorDropDown');
+  this.createDropDownFilter(this.category, '#categoryDropDown');*/
+}
+
+/*CompleteArticle.prototype.createDropDownFilter = function (filter, elementID){
+  var $options = $(elementID).children();
+  var repeat = false;
+
+  $options.each(function() {
+          ///console.log("THIS: " + $(this));
+          ///console.log('This is options.val: ' + $(this).val());
+    if($(this).val() === filter ){
+      repeat = true;
+    }
+  });
+  if(!repeat){
+    var $newOptionClone = $(elementID +' :first').clone();
+    $newOptionClone.text(filter);
+    $(elementID + ' :last').after($newOptionClone);
+  }
+};*/
```

Comment:
Avoid committing commented out code into your master branch. If it's not being used to clarify your working code, then it just adds clutter. This should be removed before making a pull request. You can always come back to an earlier commit if you want to see this code again.